app/api/v1/account_endpoint.py

```
@@ -0,0 +1,16 @@
+from fastapi_cache.decorator import cache
+
+from app.core import settings
+from app.core.cache import cache_key_builder
+from app.services.services import validate_api_key
+
+router = APIRouter(prefix="/account", tags=["account"])
+
+
+@router.get("/", summary="Account summary", response_description="Account details")
+@cache(expire=settings.CACHE_TTL_NORMAL_SECONDS, namespace="account", key_builder=cache_key_builder)
+async def account_details(api_key_data: dict = Depends(validate_api_key)):
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)